demos/common/cpp/utils/include/utils/performance_metrics.hpp

```diff
@@ -83,4 +89,4 @@ class PerformanceMetrics {
     bool firstFrameProcessed;
 };
 
-void printStagesLatency(double readLat, double preprocLat, double inferLat, double postprocLat, double renderLat);
+void logLatencyPerStage(double readLat, double preprocLat, double inferLat, double postprocLat, double renderLat);
```

Comment:
I reccomend to change it to:
```
void logLatencyPerStage(std::map<std::string, const PerformanceMetrics> metrics);
```
It would be much agile.

Reply:
You must be kidding
https://github.com/openvinotoolkit/open_model_zoo/pull/2643#discussion_r680874066

Reply:
Fine, I'll do it by myself.